HW03/Tatsiana_Zenkovich/tatsiana_zenkovich_hw_03_t_01/mentor.rb

```diff
@@ -0,0 +1,35 @@
+class Mentor
+  attr_reader :name, :surname, :students, :tasks, :notifications
+  def initialize(name, surname)
+    @name = name
+    @surname = surname
+    @notifications = []
+    @students = []
+    @tasks = []
+  end
+
+  def subscribe_to(student)
+    @notifications.clear
+    @students << student
+  end
+
+  def unsubscribe_from(student)
+    @students.delete(student)
+  end
+
+  def read_notifications!
+    @notifications = @notifications.clear
+  end
```

Comment:
Should work by this way 
```suggestion
  def read_notifications!
     @notifications.clear
  end
```

Reply:
corrected